upgrade-instructions.md

```diff
@@ -209,24 +208,6 @@ blob.export.linshare.technical.account.password
 ```
 
 -The legacy property `blob.export.linshare.token` will not used anymore, you can remove it.
-### Hybrid blobStore replaces Union blobStore
```

Comment:
that's strange to remove a note from the upgrade instructions... maybe we can just add that it has been removed later?